settings.gradle.kts

```diff
@@ -1,3 +1,5 @@
 rootProject.name = "mps-gradle-plugin"
 include("execute-generators")
-include("project-loader")
\ No newline at end of file
+include("project-loader")
+
+enableFeaturePreview("STABLE_PUBLISHING")
```

Comment:
I think we can remove this line. The spec clains we do not need them any longer: 
see: https://docs.gradle.org/current/userguide/upgrading_version_4.html#changes_5.0